epochStart/bootstrap/process.go

```diff
@@ -285,6 +268,21 @@ func (e *epochStartBootstrap) Bootstrap() (Parameters, error) {
 	return e.requestAndProcessing()
 }
 
+func (e *epochStartBootstrap) computeIfCurrentEpochIsSaved() bool {
+	e.initializeFromLocalStorage()
+	if !e.baseData.storageExists {
+		return false
+	}
+
+	computedRound := e.rounder.Index()
+	if computedRound-e.baseData.lastRound < roundGracePeriod {
```

Comment:
Is this condition safe, we really need it?

Reply:
the condition is safe. I just added if for any case the round is relatively close to one in the DB there is no reason to start syncing.

Reply:
Agree, but probably would be catch anyway by the next computation